pages/form/returning-members.js

```diff
@@ -1,17 +1,21 @@
-import React, { useState, useContext, useEffect, useRef } from 'react';
-import { isEmail } from 'validator';
+import React, { useState, useContext } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useRouter } from "next/router";
 import { ThemeContext } from 'styled-components';
-import PageTemplate from '../frontend/components/templates/PageTemplate';
+import PageTemplate from '../../frontend/components/templates/PageTemplate';
 import {
-  SystemComponent,
-  SystemSpan,
-} from '../frontend/components/atoms/SystemComponents';
-import Subtitle from '../frontend/components/atoms/Subtitle';
-import Card from '../frontend/components/atoms/Card';
-import FieldSection from '../frontend/components/molecules/Form/FieldSection';
-import Button from '../frontend/components/atoms/Button';
+  SystemComponent
+} from '../../frontend/components/atoms/SystemComponents';
+import Card from '../../frontend/components/atoms/Card';
+import FieldSection from '../../frontend/components/molecules/Form/FieldSection';
+import FormHeader from '../../frontend/components/molecules/Form/FormHeader';
+import FormFooter from '../../frontend/components/molecules/Form/FormFooter';
 
-const SCHOOL_TERM_LABELS = [
+import {validateField, clearErrorMessages, isInvalidPhoneNumber, isInvalidStudentId} from '../../frontend/util'
+
+import { updateUser } from "../../frontend/store/reducers/userReducer";
+
+const SCHOOL_TERMS = [
   '1A Co-op',
```

Comment:
Considering that some people join Waterloop before even entering into 1A, I guess it would make sense to include `1A Study` too. Chawthri is an example :) 

The original form creators purposely didn't include this options for the Returning Members form.

Reply:
Addressed. Please check.